process/templates/SIG_NAME_graduation_stage.md

```diff
@@ -1,5 +1,31 @@
 ## Special Interest Group (SIG) graduation
 
 The governing body must agree that the SIG has completed a major deliverable
-
   * Link to relevant documentation.
+
+## SIG has met all Incubating requirements
+  * "link to Incubating PR"
+
+### Governance
+SIG must have documented project governance and be able to demonstrate that governance in action.
+  * "link to governance documents"
+SIG has met at least 4 times over a period of at least 2 months since becoming incubating
+  * "link to meeting agenda"
+Have a define roadmap and annual goals for the project
+  * "link to roadmap and goals"
+
```

Comment:
```suggestion
SIG has completed a major deliverable
	  * "link to completed deliverable"

```

Roadmaps are fine, but if you're applying to be graduated I think you need to have demonstrated a major deliverable (from the SIG Lifecycle doc).